velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -353,6 +353,21 @@ TEST_F(ArithmeticTest, log1p) {
   EXPECT_TRUE(std::isnan(log1p(kNan).value_or(0)));
 }
 
+TEST_F(ArithmetricTest, expm1) {
+  static const auto expm1 = [&](std::optional<double> a) {
+    return evaluateOnce<double>("expm1(c0)", a);
+  }
+
+  const double kE = std::exp(1);
+
+  EXPECT_EQ(expm1(0), 0);
+  EXPECT_EQ(expm1(1), kE - 1);
+  // As this is only for high accuracy of little number, this is just an example
```

Comment:
Could you revise this comment a bit? I feel hard to understand. Thanks.

Reply:
I just pick a the little number 1e-12, while other value is also OK.
So this is just the example. I've added more comments here.